public/docs/ts/latest/guide/architecture.jade

```diff
@@ -92,21 +92,19 @@ figure
 
```

Comment:
Added. I had similar thoughts on the " - " after reading the guidelines. You make a good point about the colon.